img/profile.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#fff"><path d="M12,4A4,4 0 0,1 16,8A4,4 0 0,1 12,12A4,4 0 0,1 8,8A4,4 0 0,1 12,4M12,14C16.42,14 20,15.79 20,18V20H4V18C4,15.79 7.58,14 12,14Z" /></svg>
```

Comment:
did you create this or do we need license attribution?

Reply:
As far as I remember, I took it from Material Design Icons, i dnk what's needed.

Reply:
Licenses like MIT require you to attribute the authors. Find the sources and check the license. If attribution is required we can add a license file like https://github.com/nextcloud/calendar/blob/main/img/LICENSES.md

Reply:
@ChristophWurst The icon of profile is from material-design-icons: https://pictogrammers.com/library/mdi/icon/account/

What attribution do we need here? All projects here are under https://pictogrammers.com/docs/general/license/ Pictogrammers Free License, where Icons marked under Apache 2.0.

Reply:
Licenses like MIT and APache 2.0 require you to attribute the authors. We can add a license file like https://github.com/nextcloud/calendar/blob/main/img/LICENSES.md

https://law.stackexchange.com/questions/6048/does-the-apache-2-0-license-require-attribution-on-a-commercial-work-on-which-it